[DO NOT MERGE][Visual Refresh] Input design changes

mgadewoll opened the pull request:
## Summary

>[!IMPORTANT]
DO NOT MERGE.
This is a testing PR for the Visual Refresh input design changes.

This PR introduces visual style changes to input and form layout elements as part of the Visual Refresh project to modernize the UI of our product.

>[!NOTE]
💻 [Deployment](https://kibana-pr-227054.kb.us-west2.gcp.elastic-cloud.com)

## Changes

Almost all changes were done on EUI side. The changes to form elements were done in https://github.com/elastic/eui/pull/8767 and https://github.com/elastic/eui/pull/8778.

- add newly introduced prop `colorModes` on **EuiSelectableTemplateSitewide** (https://github.com/elastic/eui/pull/8806) to ensure expected output
- update color values in tests/snapshots

### What changes visibly?

🔗 See a more detailed overview of changes [here](https://docs.google.com/document/d/1pxfev-NQhvbyMCkIGDNgEC4XewnkV_N32p8aEyhvRHo/edit?usp=sharing).

The visually most noticeable change is the update of state indicators for focus and invalid. The indicator underline was changed to a full outline.

| before | after |
|---|---|
| ![Screenshot 2025-06-24 at 10 11 54](https://github.com/user-attachments/assets/7773bc67-8a8a-4099-93b7-5ac3d3a84515) | ![Screenshot 2025-06-24 at 10 11 46](https://github.com/user-attachments/assets/8ceb9cfc-b3dc-4f35-b2ba-3b879cf14ba3) |
| ![Screenshot 2025-06-24 at 10 12 47](https://github.com/user-attachments/assets/95e7a20c-af0e-4015-a2cd-a63662d9f8b8) | ![Screenshot 2025-06-24 at 10 12 54](https://github.com/user-attachments/assets/0038c48b-853f-401e-a753-663c18806c4a) |

1. General form layout and input element updates
    - updated text and border colors
    - visible hover states
    - updated state indicators (e.g. focus and invalid)
    - updated disabled, readonly, drag etc states
    - slightly updated paddings
    - increased icon size in compressed forms
    - added border between form control layout append/prepend elements and main form element
2. **EuiSelectableTemplateSitewide**
    - added prop `colorModes` to handle color modes for search and popover elements separately

### Which EUI components are affected?

- EuiFieldText
- EuiFieldNumber
- EuiFielPassword
- EuiFieldSearch
- EuiTextarea
- EuiSelect
- EuiSuperSelect
- EuiFilePicker
- EuiDatePicker
- EuiSuperDatePicker
- EuiComboBox
- EuiFormLabel
- EuiFormErrorText
- EuiFormControlLayout
- EuiFormControlLayoutDelimited
- EuiFormControlLayoutIcons
- EuiSelectableTemplateSitewide

## Screenshots

<details>
<summary>View comparison screenshots</summary>

| LIGHT | DARK |
|---|---|
| ![Kapture 2025-06-23 at 15 35 36](https://github.com/user-attachments/assets/a8133fc8-950d-4544-be30-2f8406054690) | ![Kapture 2025-06-23 at 15 40 44](https://github.com/user-attachments/assets/92b323fa-c237-4089-98db-1eb893fe4bfe) |
| ![Kapture 2025-06-23 at 14 54 52](https://github.com/user-attachments/assets/2ff4c719-13f9-43f3-b325-45d15994c379) | ![Kapture 2025-06-23 at 14 53 26](https://github.com/user-attachments/assets/39a414b9-615b-48b1-b7f3-d6f81979b214) |
| ![Kapture 2025-06-23 at 14 58 11](https://github.com/user-attachments/assets/66885b1d-e71a-46b7-95dc-4864779d7528) | ![Kapture 2025-06-23 at 15 31 53](https://github.com/user-attachments/assets/6475d86d-0da6-4e49-8f94-9281ba764554) |
| ![Kapture 2025-06-23 at 15 33 09](https://github.com/user-attachments/assets/43aa099d-348c-4e48-a276-9f76379d7beb) | ![Kapture 2025-06-23 at 15 32 41](https://github.com/user-attachments/assets/bd20d6f0-7614-44b9-9a3d-4187b870e3cf) |
| ![Kapture 2025-06-23 at 15 49 14](https://github.com/user-attachments/assets/7cdbc44d-7d1c-4978-9473-c886b64994d5) | ![Kapture 2025-06-23 at 15 48 47](https://github.com/user-attachments/assets/a074b827-e786-4696-ac54-21287ef053fd) |

</details>

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309779)
* Commit: bf2d507a068515740130c281e5ec035425a05bad

### Failed CI Steps
* [Check Types](https://buildkite.com/elastic/kibana-pull-request/builds/309779#01978247-8b95-46db-873b-384616ab17d0)

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309779","number":309779,"commit":"bf2d507a068515740130c281e5ec035425a05bad"}],"number":309779}
buildkite-pr-comment-->

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/248

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/251

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/309998)
* Commit: b153c366eb951a62e957a1371d8f7bf08b63e2e5

### Failed CI Steps
* [Build and Deploy to Cloud](https://buildkite.com/elastic/kibana-pull-request/builds/309998#0197835a-f10b-4dde-b134-a2f71d4b686e)

### History
* :broken_heart: [Build #309779](https://buildkite.com/elastic/kibana-pull-request/builds/309779) failed bf2d507a068515740130c281e5ec035425a05bad

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309998","number":309998,"commit":"b153c366eb951a62e957a1371d8f7bf08b63e2e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309779","number":309779,"commit":"bf2d507a068515740130c281e5ec035425a05bad"}],"number":309998}
buildkite-pr-comment-->

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/251

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310378)
* Commit: e06749276a1958447ace0de7f383f793e067e9eb

### Failed CI Steps
* [Build Kibana Distribution](https://buildkite.com/elastic/kibana-pull-request/builds/310378#019787c9-8b8a-4df0-a7c0-0e4f0cacec7b)

### History
* :broken_heart: [Build #309998](https://buildkite.com/elastic/kibana-pull-request/builds/309998) failed b153c366eb951a62e957a1371d8f7bf08b63e2e5
* :broken_heart: [Build #309779](https://buildkite.com/elastic/kibana-pull-request/builds/309779) failed bf2d507a068515740130c281e5ec035425a05bad

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310378","number":310378,"commit":"e06749276a1958447ace0de7f383f793e067e9eb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309998","number":309998,"commit":"b153c366eb951a62e957a1371d8f7bf08b63e2e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309779","number":309779,"commit":"bf2d507a068515740130c281e5ec035425a05bad"}],"number":310378}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310401)
* Commit: 7cee1b8c77e1204eb29146a2de1c53509422a8c1

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/310401#019787ff-6bb6-49de-a3f8-6400ed37d68a)

### History
* :broken_heart: [Build #310378](https://buildkite.com/elastic/kibana-pull-request/builds/310378) failed e06749276a1958447ace0de7f383f793e067e9eb
* :broken_heart: [Build #309998](https://buildkite.com/elastic/kibana-pull-request/builds/309998) failed b153c366eb951a62e957a1371d8f7bf08b63e2e5
* :broken_heart: [Build #309779](https://buildkite.com/elastic/kibana-pull-request/builds/309779) failed bf2d507a068515740130c281e5ec035425a05bad

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310401","number":310401,"commit":"7cee1b8c77e1204eb29146a2de1c53509422a8c1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310378","number":310378,"commit":"e06749276a1958447ace0de7f383f793e067e9eb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309998","number":309998,"commit":"b153c366eb951a62e957a1371d8f7bf08b63e2e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309779","number":309779,"commit":"bf2d507a068515740130c281e5ec035425a05bad"}],"number":310401}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310406)
* Commit: 9d57b95bf950f3fa792f10259f965e1b98177506
* [Cloud Deployment](https://kibana-pr-224382.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #310401](https://buildkite.com/elastic/kibana-pull-request/builds/310401) failed 7cee1b8c77e1204eb29146a2de1c53509422a8c1
* :broken_heart: [Build #310378](https://buildkite.com/elastic/kibana-pull-request/builds/310378) failed e06749276a1958447ace0de7f383f793e067e9eb
* :broken_heart: [Build #309998](https://buildkite.com/elastic/kibana-pull-request/builds/309998) failed b153c366eb951a62e957a1371d8f7bf08b63e2e5
* :broken_heart: [Build #309779](https://buildkite.com/elastic/kibana-pull-request/builds/309779) failed bf2d507a068515740130c281e5ec035425a05bad

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310406","number":310406,"commit":"9d57b95bf950f3fa792f10259f965e1b98177506"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310401","number":310401,"commit":"7cee1b8c77e1204eb29146a2de1c53509422a8c1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310378","number":310378,"commit":"e06749276a1958447ace0de7f383f793e067e9eb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309998","number":309998,"commit":"b153c366eb951a62e957a1371d8f7bf08b63e2e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309779","number":309779,"commit":"bf2d507a068515740130c281e5ec035425a05bad"}],"number":310406}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/310527)
* Commit: 2667962555c1f86d13b2aa92b84566005e518399
* [Cloud Deployment](https://kibana-pr-224382.kb.us-west2.gcp.elastic-cloud.com)

### History
* :broken_heart: [Build #310406](https://buildkite.com/elastic/kibana-pull-request/builds/310406) failed 9d57b95bf950f3fa792f10259f965e1b98177506
* :broken_heart: [Build #310401](https://buildkite.com/elastic/kibana-pull-request/builds/310401) failed 7cee1b8c77e1204eb29146a2de1c53509422a8c1
* :broken_heart: [Build #310378](https://buildkite.com/elastic/kibana-pull-request/builds/310378) failed e06749276a1958447ace0de7f383f793e067e9eb
* :broken_heart: [Build #309998](https://buildkite.com/elastic/kibana-pull-request/builds/309998) failed b153c366eb951a62e957a1371d8f7bf08b63e2e5
* :broken_heart: [Build #309779](https://buildkite.com/elastic/kibana-pull-request/builds/309779) failed bf2d507a068515740130c281e5ec035425a05bad

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310527","number":310527,"commit":"2667962555c1f86d13b2aa92b84566005e518399"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310406","number":310406,"commit":"9d57b95bf950f3fa792f10259f965e1b98177506"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310401","number":310401,"commit":"7cee1b8c77e1204eb29146a2de1c53509422a8c1"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/310378","number":310378,"commit":"e06749276a1958447ace0de7f383f793e067e9eb"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309998","number":309998,"commit":"b153c366eb951a62e957a1371d8f7bf08b63e2e5"},{"buildStatus":{"state":"failed","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/309779","number":309779,"commit":"bf2d507a068515740130c281e5ec035425a05bad"}],"number":310527}
buildkite-pr-comment-->

mgadewoll committed:
chore: update eui packages

- includes visual refresh input updates

mgadewoll committed:
refactor: adds colorModes prop to handle global search color modes

mgadewoll committed:
refactor: update JSON import path

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/257

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/258

mgadewoll committed:
test: update tests and snapshots

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/267

jughosta reviewed (commented):
Noticed some issues in Data View flyout in error state:
 
- the first input does not get red outline (same on main)
- the outline for the second input is missing smooth round borders

<img width="926" alt="Screenshot 2025-07-04 at 13 47 48" src="https://github.com/user-attachments/assets/9be21f56-904b-4867-a4fa-886529f072d2" />

jughosta commented on x-pack/test_serverless/functional/test_suites/observability/discover/context_awareness/_get_cell_renderers.ts:
```diff
@@ -62,7 +62,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
           expect(await logLevelBadge.getVisibleText()).to.be('debug');
           expect(await logLevelBadge.getComputedStyle('background-color')).to.be(
-            'rgba(189, 215, 255, 1)'
+            'rgba(232, 241, 255, 1)'
```
We might want to keep the previous colors (hard code them?) in `src/platform/packages/shared/kbn-discover-utils/src/data_types/logs/utils/get_log_level_color.ts` for time being since we are discussing to change them in https://github.com/elastic/kibana/issues/225651

jughosta replied:
Hi @mgadewoll,
Thanks for looking into it! It appears on Discover only in Observability space.
This color is used for the row color indicators (on the left side of each row) and for `log.level` badge color. 

<img width="1354" alt="Screenshot 2025-07-07 at 09 49 54" src="https://github.com/user-attachments/assets/e376a2f3-d0ab-4ba7-bd60-ca9580db0e60" />

mgadewoll replied:
>This grey does not provide enough contrast in comparison with the previous value.

@jughosta That's not a change that was introduced here. (it only surfaces because the tests had old values).
ℹ️ `EuiDataGrid` has recently received an update to the hover styling which wasn't added in this current EUI test release yet (I'll update this branch in a bit).

With that it will look like this:
![Screenshot 2025-07-07 at 14 07 09](https://github.com/user-attachments/assets/4324dc80-adc3-430c-ac0b-63eb4298b66f)

We would suggest to use the subdued badge colors for the "DEBUG" log level instead ([code](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-discover-utils/src/data_types/logs/utils/get_log_level_color.ts#L30)): `euiTheme.colors.backgroundLightText`.  

![Screenshot 2025-07-07 at 14 07 19](https://github.com/user-attachments/assets/7886d85a-0c7c-4a7a-b5c3-beea4fa4d975)

cc @ek-so

jughosta replied:
Either is okay with me, thanks!

mgadewoll replied:
We won't update this here now as it's not related to the input changes. We'll leave it to the owning team to check if the suggestion works better 🙂

mgadewoll committed:
chore: update eui packages

- includes fix for invalid border corners being clipped

mgadewoll committed:
fix: update token to ensure filter button group border is rendered

mgadewoll committed:
refactor: update inTableSearchControl styles

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
PR Cloud deployment started at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/272

jughosta commented:
> This is an issue on implementation side and it's not introduced by the input changes. The issue is that only the EuiFormRow has the inValid prop but not the EuiFieldText ([code](https://github.com/elastic/kibana/blob/main/src/platform/plugins/shared/data_view_editor/public/components/form_fields/name_field.tsx#L80)). For both to be correctly styled, both require the prop to be set ([docs](https://eui.elastic.co/docs/components/forms/validation/))

Thanks! Addressing in https://github.com/elastic/kibana/pull/226822

mgadewoll committed:
test: update tests and snapshots

- triggered by updated initial option for EuiColorVisStore to ensure Borealis values

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/316386)
* Commit: fd001dc34c16c6a220344b23d68503bafc2d68a4

### Failed CI Steps
* [Serverless Detection Engine - Exceptions - Security Solution Cypress Tests #1](https://buildkite.com/elastic/kibana-pull-request/builds/316386#0197e570-a201-4e0a-b968-c2429df4bee7)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316386#0197e570-a201-4e0a-b968-c2429df4bee7) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316386/jobs/0197e570-a201-4e0a-b968-c2429df4bee7/artifacts/0197e5ac-5aa2-460f-85c6-1b83789cfb38) Serverless Detection Engine - Exceptions - Security Solution Cypress Tests #<span></span>1 / Auto populate exception with Alert data Should create a Rule exception from Alerts take action button and change multiple exception items without resetting to initial auto-prefilled entries Should create a Rule exception from Alerts take action button and change multiple exception items without resetting to initial auto-prefilled entries

### History
* :broken_heart: [Build #316307](https://buildkite.com/elastic/kibana-pull-request/builds/316307) failed 1fe2718a6a027261f171328c6eb050abc0f0457c
* :green_heart: [Build #312146](https://buildkite.com/elastic/kibana-pull-request/builds/312146) succeeded 6c2f6b6ec3b79db8b60594d574b308d6f7a7da5a
* :broken_heart: [Build #312042](https://buildkite.com/elastic/kibana-pull-request/builds/312042) failed c45ed6040883ffeb8b52701d023db5361fbc72a0
* :broken_heart: [Build #311961](https://buildkite.com/elastic/kibana-pull-request/builds/311961) failed 52a3f0788ad44a4ebf1eab3e406af1f5a6c8b269
* :broken_heart: [Build #311883](https://buildkite.com/elastic/kibana-pull-request/builds/311883) failed 3c485c91cf938e95de73afe5c3a78a499170ce0d

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316386","number":316386,"commit":"fd001dc34c16c6a220344b23d68503bafc2d68a4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316307","number":316307,"commit":"1fe2718a6a027261f171328c6eb050abc0f0457c"},{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312146","number":312146,"commit":"6c2f6b6ec3b79db8b60594d574b308d6f7a7da5a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312042","number":312042,"commit":"c45ed6040883ffeb8b52701d023db5361fbc72a0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311961","number":311961,"commit":"52a3f0788ad44a4ebf1eab3e406af1f5a6c8b269"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311883","number":311883,"commit":"3c485c91cf938e95de73afe5c3a78a499170ce0d"}],"number":316386}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/316594)
* Commit: 64235d8d051f318df113e4be151eb1b94febc77d

### Failed CI Steps
* [Detection Engine - Exceptions - Security Solution Cypress Tests #3](https://buildkite.com/elastic/kibana-pull-request/builds/316594#0197e8ea-b77d-4cf7-a021-188ad09d5635)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/316594#0197e8ea-b77d-4cf7-a021-188ad09d5635) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/316594/jobs/0197e8ea-b77d-4cf7-a021-188ad09d5635/artifacts/0197e91d-ab28-4ddf-8361-17ac600e0126) Detection Engine - Exceptions - Security Solution Cypress Tests #<span></span>3 / Auto populate exception with Alert data Should create a Rule exception from Alerts take action button and change multiple exception items without resetting to initial auto-prefilled entries Should create a Rule exception from Alerts take action button and change multiple exception items without resetting to initial auto-prefilled entries

### History
* :broken_heart: [Build #316386](https://buildkite.com/elastic/kibana-pull-request/builds/316386) failed fd001dc34c16c6a220344b23d68503bafc2d68a4
* :broken_heart: [Build #316307](https://buildkite.com/elastic/kibana-pull-request/builds/316307) failed 1fe2718a6a027261f171328c6eb050abc0f0457c
* :green_heart: [Build #312146](https://buildkite.com/elastic/kibana-pull-request/builds/312146) succeeded 6c2f6b6ec3b79db8b60594d574b308d6f7a7da5a
* :broken_heart: [Build #312042](https://buildkite.com/elastic/kibana-pull-request/builds/312042) failed c45ed6040883ffeb8b52701d023db5361fbc72a0
* :broken_heart: [Build #311961](https://buildkite.com/elastic/kibana-pull-request/builds/311961) failed 52a3f0788ad44a4ebf1eab3e406af1f5a6c8b269

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316594","number":316594,"commit":"64235d8d051f318df113e4be151eb1b94febc77d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316386","number":316386,"commit":"fd001dc34c16c6a220344b23d68503bafc2d68a4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316307","number":316307,"commit":"1fe2718a6a027261f171328c6eb050abc0f0457c"},{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312146","number":312146,"commit":"6c2f6b6ec3b79db8b60594d574b308d6f7a7da5a"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/312042","number":312042,"commit":"c45ed6040883ffeb8b52701d023db5361fbc72a0"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/311961","number":311961,"commit":"52a3f0788ad44a4ebf1eab3e406af1f5a6c8b269"}],"number":316594}
buildkite-pr-comment-->

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
PR Project deployment started at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/504

tkajtoch commented:
/oblt-deploy

mgadewoll committed:
Merge branch 'main' into eui/324-input-visual-refresh

kibanamachine commented:
<!-- kbn-message-context:project-deploy-job -->
PR Project deployment started at: https://buildkite.com/elastic/kibana-deploy-project-from-pr/builds/505

mgadewoll commented on src/platform/packages/shared/kbn-data-grid-in-table-search/src/__snapshots__/in_table_search_input.test.tsx.snap:
```diff
@@ -120,7 +120,7 @@ exports[`InTableSearchInput renders input when loading 1`] = `
         <span
           aria-label="Loading"
-          class="euiLoadingSpinner emotion-euiLoadingSpinner-s"
+          class="euiLoadingSpinner emotion-euiLoadingSpinner-m"
```
@jughosta Fyi, I added small updates to improve the mentioned UI issues ([deployment](https://kibana-pr-227054.kb.us-west2.gcp.elastic-cloud.com))

![Screenshot 2025-07-07 at 14 52 07](https://github.com/user-attachments/assets/bc8991c6-3e2a-4f29-b3f7-57e93261107f)

![Screenshot 2025-07-07 at 14 52 21](https://github.com/user-attachments/assets/a2e73ca7-5a42-48b7-bca0-97b0dbaa59b3)

kibanamachine commented:
<!-- kbn-message-context:cloud-deploy-job -->
Cloud deployment initiated, see credentials at: https://buildkite.com/elastic/kibana-deploy-cloud-from-pr/builds/273

mgadewoll committed:
test: ensure click fires

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/317385)
* Commit: ba084f290be949238058f261f553bb4ad6280b0b
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-224382-ba084f290be9`

### History
* :broken_heart: [Build #317161](https://buildkite.com/elastic/kibana-pull-request/builds/317161) failed c5370ae94f8f053ab49af5cad7b5b3446f13c0e7
* :broken_heart: [Build #316821](https://buildkite.com/elastic/kibana-pull-request/builds/316821) failed ebe3d9bb2e9d892dc30c33842781d276008fd618
* :broken_heart: [Build #316594](https://buildkite.com/elastic/kibana-pull-request/builds/316594) failed 64235d8d051f318df113e4be151eb1b94febc77d
* :broken_heart: [Build #316386](https://buildkite.com/elastic/kibana-pull-request/builds/316386) failed fd001dc34c16c6a220344b23d68503bafc2d68a4
* :broken_heart: [Build #316307](https://buildkite.com/elastic/kibana-pull-request/builds/316307) failed 1fe2718a6a027261f171328c6eb050abc0f0457c

cc @mgadewoll

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317385","number":317385,"commit":"ba084f290be949238058f261f553bb4ad6280b0b"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/317161","number":317161,"commit":"c5370ae94f8f053ab49af5cad7b5b3446f13c0e7"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316821","number":316821,"commit":"ebe3d9bb2e9d892dc30c33842781d276008fd618"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316594","number":316594,"commit":"64235d8d051f318df113e4be151eb1b94febc77d"},{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316386","number":316386,"commit":"fd001dc34c16c6a220344b23d68503bafc2d68a4"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/316307","number":316307,"commit":"1fe2718a6a027261f171328c6eb050abc0f0457c"}],"number":317385}
buildkite-pr-comment-->
